packages/beacon-node/src/chain/archiver/interface.ts

```diff
@@ -0,0 +1,46 @@
+import {CheckpointWithHex} from "@lodestar/fork-choice";
+import {Metrics} from "../../metrics/metrics.js";
+import {RootHex} from "@lodestar/types";
+
+export enum ArchiveMode {
+  Full = "full",
```

Comment:
If you think of a better name for current archiver strategy we use, please suggest.